scala-generator/src/main/scala/models/http4s/Http4sClient.scala

```diff
@@ -65,7 +65,9 @@ ${headerString.indent(6)}
         apiHeaders ++
         defaultHeaders ++
         requestHeaders
-      ).toList.map { case (k, v) => org.http4s.Header(k, v) })
+      ).toList
+        .groupBy(_._1).map{ case (headerName, duplicatesValList) => (headerName, duplicatesValList.last._2) }.toList
```

Comment:
Just a nit, but this feels a bit terser (ignoring the `headerName` and just using the last of the values): `.groupBy(_._1).map { case (_, l) => l.last }.toList`

Reply:
Or even better, combine with the second map (and drop the duplicate `.toList`):

```
val headers = org.http4s.Headers((
        apiHeaders ++
        defaultHeaders ++
        requestHeaders
      ).groupBy(_._1).map { case (k, l) => org.http4s.Header(k, l.last._2) }.toList
```

Reply:
Done.
Much cleaner indeed.